src/components/MessageList/VirtualizedMessageList.js

```diff
@@ -125,27 +152,14 @@ const VirtualizedMessageList = ({
         totalCount={messages.length}
         overscan={overscan}
         followOutput={true}
-        maxHeightCacheSize={2000} // reset the cache once it reaches 2k
-        scrollSeek={scrollSeekPlaceHolder}
```

Comment:
Hmm, so I updated the `components` prop to include the `scrollSeekPlaceholder`, but the `ScrollSeekConfiguration` is new. Should we allow SDK users to override this virtuoso prop? Should we provide a default value?

Reply:
we don't have a default one for now, I'd just check and break the user passed prop into UI and config until we go with a major release and update the API

Reply:
I've updated the code now, thanks for pointing this out because I didn't realise there was so much going on in that prop. Somehow the typescript definitions also got a little messed up so tsc didn't complain that I missed a bunch of stuff there. Should be better now, but I do think the code looks a bit ugly. Definitely something we'll want to fix the next time we make breaking changes.

Reply:
awesome thanks, can you also leave a todo comment so we won't forget about it later

Reply:
Yeah, I added this to the component:
```
// TODO: split scrollSeekPlaceholder into two props (e.g. ScrollSeekPlaceholder and scrollSeekConfiguration) when making breaking changes
```

and this to the typedefs:
```
*  Note: virtuoso has broken out the placeholder value and instead includes it in its components prop.
*  TODO: break out placeholder when making other breaking changes.
```